content/en/releases/checklist-for-victims-of-a-data-breach.md

```diff
@@ -0,0 +1,187 @@
+
+### Request prohibition for address change with Posti free-of-charge
+
+*   You can prevent address changes by activating [address change protection from Posti](https://www.posti.fi/fi/asiakastuki/tiedotteet/postin-ohje-psykoterapiakeskus-vastaamon-tietomurtoon-liittyen-nain-teet). (currently only in Finnish)
+*   The ban is requested from [Posti's online customer service portal](https://www.posti.fi/omatpalvelut/feedback?lang=fi) under "Address change protection, Feedback about Mail forwarding or Nearby Mailbox."
+*   The protection prevents address changes and mail delivery suspensions as long as the protection is enabled. A criminal can't then order deliveries or invoices under your name to another address.
+*   [Posti's hints on address changes and improving one's data security](https://www.posti.fi/fi/asiakastuki/tiedotteet/20200615_muuttoilmoitus_ja_tietoturva) (currently only in Finnish)
+
+### Request a registration ban from Finnish Patent and Registration Office (PRH) free-of-charge
+
+*   A registration ban will prevent fraud where you are registered as a responsible person for some company. When you request a registration prohibition from PRH, you can't be registered as responsible for any company or registered organization.
+*   You can request the prohibition as [instructed by the Finnish Patent and Registration Office](https://www.prh.fi/fi/kaupparekisteri/valty_huijauksilta/rekisterointikielto.html) (currently only in Finnish)
+
+### Consider prohibiting passing of your information from Finnish Population Information System
+
+*   If your information has changed after the breach, it may be a good idea to protect your current contact details. This would prevent anyone from obtaining updated information on you from the system.
+*   Finnish Digital and Population Data Services Agency: [Refuse allowing the release of your data](https://dvv.fi/tietojen-luovuttamisen-kieltaminen) (currently only in Finnish)
+
+### Passwords and two-factor authentication
+
+*   Phishing and spam emails may increase and several parties my try to extort you. Cyber criminals may distribute and sell your information to other criminals and some may try guessing your passwords on different services. Make sure you have a strong password in all services and start using two-factor authentication if possible. You should also change the password of the email account which has been leaked.
+*   Instructions for creating strong passwords
+    1. The longer your password is, the safer it is. Good minimum length is 16 characters.
```

Comment:
Just:   The longer your password is, the safer it is.  We would recommend including a mix of numbers, upper and lower case letters and a symbol. 

Length of password ---->  difficult one,,  16 characters would be too much for many people

Reply:
OK, I just combined #2 with #1 .. split accordinly